packages/shared/routes/onboarding/views/backup/views/BackupToFileView.svelte

```diff
@@ -30,7 +30,11 @@
 
     async function handleSubmitClick(): Promise<void> {
         if (isStrongholdPasswordValid) {
-            await onboardingBackupFileFunction()
+            try {
+                await onboardingBackupFileFunction()
+            } catch (error) {
+                return
```

Comment:
Maybe log the error, just so we would be aware if something happens?

Reply:
It is being logged and notified in "backupInitialStronghold.ts:18"